modules/agar/src/test/ts/atomic/api/TestStoreTest.ts

```diff
@@ -0,0 +1,139 @@
+import { context, describe, it } from '@ephox/bedrock-client';
+import { Arr, Fun } from '@ephox/katamari';
+
+import * as GeneralSteps from 'ephox/agar/api/GeneralSteps';
+import { Pipeline } from 'ephox/agar/api/Pipeline';
+import { Step } from 'ephox/agar/api/Step';
+import { TestStore } from 'ephox/agar/api/TestStore';
+
+interface TestData {
+  readonly name: string;
+  readonly age: number;
+  readonly isHappy: boolean;
+}
+
+describe('atomic.agar.api.TestStoreTest', () => {
+  const strings = [ 's', 'short', 'string', 'loooooong' ];
+  const stringsSorted = Arr.sort(strings);
+
+  const numbers = [ 1, 999, 100000, 77 ];
+  const numbersSorted = Arr.sort(numbers);
+
+  const objects: TestData[] = [
+    { name: 'NN', age: 1, isHappy: true },
+    { name: 'CCCCCCC', age: 999, isHappy: false },
+    { name: 'MMM', age: 20, isHappy: true }
+  ];
+
+  const objectsSortedByName = Arr.sort(objects, (a, b) => a.name.length - b.name.length);
+  const objectsSortedByAge = Arr.sort(objects, (a, b) => a.age - b.age);
+  const objectsSortedByIsHappy = Arr.sort(objects, (a, b) => Number(a.isHappy) - Number(b.isHappy));
+
+  context('Step based', () => {
+    it('TestStore strings', () => {
```

Comment:
We should include Jira numbers as prefixes on strings so it's easy to see why something was added. This is a little bit of an edge case since it was just moved though. Up to you all if you include them on the `it` blocks.

Reply:
Thanks Lee. Happy to include the ticket numbers as prefixes on `it` blocks to make things clearer.